extensions/wikia/PageHeader/templates/PageHeaderController_actionButton.php

```diff
@@ -20,7 +20,7 @@ class="wds-is-squished wds-button"
 						   href="<?= $action['href'] ?>"
 						   data-tracking="<?= $action['data-tracking'] ?>"
 						   <?= empty( $action['accesskey'] ) ? '' : "accesskey=\"{$action['accesskey']}\"" ?>>
```

Comment:
The `accesskey` comes from a message as well and should be escaped just in case of previous customisaitons.